integrations-and-supported-tools/transformers/notebooks/Neptune_Transformers.ipynb

```diff
@@ -0,0 +1,382 @@
+   "metadata": {},
+   "source": [
+    "Create the training arguments for model finetuning"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "model_name = model_checkpoint.split(\"/\")[-1]\n",
+    "\n",
+    "args = TrainingArguments(\n",
+    "    f\"{model_name}-finetuned-{task}\",\n",
+    "    evaluation_strategy = \"epoch\",\n",
+    "    save_strategy = \"epoch\",\n",
+    "    learning_rate=2e-5,\n",
+    "    per_device_train_batch_size=batch_size,\n",
+    "    per_device_eval_batch_size=batch_size,\n",
+    "    num_train_epochs=3,\n",
+    "    weight_decay=0.01,\n",
+    "    load_best_model_at_end=True,\n",
+    "    report_to=\"none\",\n",
+    ")\n",
+    "\n",
+    "validation_key = 'validation'"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "#### Setting up the NeptuneCallback"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "We pass the `run` that was created earlier to the NeptuneCallback. The NeptuneCallback will take care of logging the metadata during the training phase. You can customize the metadata that is logged by the callback by passing additional arguments to the Callback.\n",
+    "\n",
+    "See https://docs.neptune.ai/integrations-and-supported-tools/model-training/hugging-face for more details"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Instantiate the NeptuneCallback with our `run`"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "neptune_callback = NeptuneCallback(run=run)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "trainer = Trainer(\n",
+    "    model,\n",
+    "    args,\n",
+    "    train_dataset=encoded_dataset[\"train\"],\n",
+    "    eval_dataset=encoded_dataset[validation_key],\n",
+    "    callbacks=[neptune_callback],\n",
+    "    tokenizer=tokenizer,\n",
+    ")"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "**NOTE**: One important thing to keep in mind is that `run` is stopped once `Trainer.train` is finished. Hence, we don't need to call `run.stop` explicitly (which is otherwise required when using Notebook)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "trainer.train()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "tags": [
+     "comment"
+    ]
+   },
+   "source": [
+    "## Explore the results in Neptune\n",
+    "\n",
+    "We just finetuned our model with the new data. Let's see an example of the data that was logged to Neptune.\n",
+    "\n",
+    "You can also check out an [example run](https://app.neptune.ai/o/common/org/quickstarts/e/QUI-81820/charts)."
```

Comment:
Lets use an example from [showcase/project-text-summarization-hf](https://app.neptune.ai/o/showcase/org/project-text-summarization-hf/experiments?split=tbl&dash=charts&viewId=standard-view). Also remember to tag the selected run as a showcase-run